keyboards/anavi/macropad8/readme.md

```diff
@@ -0,0 +1,18 @@
+# ANAVI Macro Pad 8
+
+
+ANAVI Macro Pad 8 is an open source mini mechanical keyboard with 8 keys, backlit, addressable RGB WS2812B LED strip on the back and mini OLED display. Powered by ATmega32U4 microcontroller and with microUSB connector.
+
+* Keyboard Maintainer: Leon Anavi
+* Hardware Supported: ANAVI Macro Pad 8
+* Hardware Availability: [PCB Files](https://github.com/AnaviTechnology/anavi-macro-pad-8)
```

Comment:
Just a heads-up, this link is currently Error 404.

Reply:
Ops, sorry, I was a private repo, I made it public. Thanks, Leon